File: Lab-1/Project-1/project-1.cpp

#define LED_GREEN  2
#define LED_BLUE  13
#define LED_RED  15

void setup() {
  pinMode(LED_GREEN, OUTPUT);
  pinMode(LED_BLUE, OUTPUT);
  pinMode(LED_RED, OUTPUT);
}

void loop() {
    int leds[] = {LED_GREEN, LED_BLUE, LED_RED};
   
    for (int i = 0; i < 3; i++) {
      digitalWrite(leds[i], HIGH);
      delay(1000); 
      digitalWrite(leds[i], LOW);
      delay(1000); 
    }
}

File: Lab-1/Project-1/project-1.py
from time import sleep_ms
from machine import Pin

ledGreen=Pin(2,Pin.OUT)
ledBlue=Pin(15,Pin.OUT)
ledRed=Pin(18,Pin.OUT)
arr = [ledBlue, ledGreen, ledRed]

try:
    while True:
        for element in arr:
            element.value(1)
            sleep_ms(200)
            element.value(0)

except KeyboardInterrupt:
    pass

